pkg/controller/virtinformers.go

```diff
@@ -60,7 +60,13 @@ import (
 )
 
 const (
-	OperatorLabel = kubev1.ManagedByLabel + "=" + kubev1.ManagedByLabelOperatorValue
+	/*
+		TODO: replace the assignment after few releases (after 0.47)
+		The new assignment is to avoid error on update
+		(operator can't recognize components with the old managed-by label's value)
+	*/
+	//OperatorLabel = kubev1.ManagedByLabel + "=" + kubev1.ManagedByLabelOperatorValue
+	OperatorLabel = kubev1.ManagedByLabel + " in (" + kubev1.ManagedByLabelOperatorValue + ",kubevirt-operator )"
```

Comment:
I'd take this chance to rename this misleadingly named variable, since it doesn't represent a label but rather a label selector. `OperatorLabelSelector`? Also recommend to use `fmt.Sprintf` here since it'll result with more readable (and less prone to whitespace/parentheses mistakes):
```suggestion
OperatorLabelSelector = fmt.Sprintf("%s in (%s,%s)", 
	kubev1.ManagedByLabel,
	kubev1.ManagedByLabelOperatorValue,
	kubev1.ManagedByLabelOperatorValueOld)
```

Reply:
Hey Zvi,
Thanks for the note.
Actually i already tried to do that but it's not possible because this code is inside a `const{..}` block
and fmt.Sprintf doesn't return const value and in my opinion it's better to keep OperatorLabelSelector const
than make this code more readable.

Reply:
You're right, I missed that. Converting to `var` is quite a common practice for defining runtime-evaluated "constants" (here, the constness of it is only by convention). Another option to achieve constness is by defining it as a function. Either way this isn't a big thing, and can be left as is too (I'd still rename the variable though ;-) )